lib/phobos_db_checkpoint.rb

```diff
@@ -20,48 +20,69 @@ module PhobosDBCheckpoint
 
   class << self
     attr_reader :db_config
-    attr_accessor :db_config_path, :db_dir, :migration_path
+    attr_accessor :db_config_path, :db_dir, :migration_path, :verbose
 
     # :nodoc:
     # ActiveRecord hook
     def table_name_prefix
       :phobos_db_checkpoint_
     end
 
-    def configure(pool_size: nil)
-      load_db_config(pool_size: pool_size)
+    def configure(options = {})
+      warn Kernel.caller.first + "[DEPRECATION] options are deprecated, use configuration files instead" if options.keys.any?
```

Comment:
Why not do string interpolation? Then it would be visible that there's no space between `Kernel.caller.first` and `[DEPRECATION]` text.